.circleci/config.yml

```diff
@@ -0,0 +1,267 @@
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+  build-exampleDataTrack:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleDataTrack
+          command: ./gradlew -q exampleDataTrack:assemble
+
+      # Save the builds to workspace
+      - persist_to_workspace:
+          root: .
+          paths:
+            - exampleDataTrack/build
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+
+  build-exampleScreenCapturer:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleScreenCapturer
+          command: ./gradlew -q exampleScreenCapturer:assemble
+
+      # Save the builds to workspace
+      - persist_to_workspace:
+          root: .
+          paths:
+            - exampleScreenCapturer/build
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+  build-exampleVideoInvite:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleVideoInvite
+          command: ./gradlew -q exampleVideoInvite:assemble
+
+      # Save the builds to workspace
+      - persist_to_workspace:
+          root: .
+          paths:
+            - exampleVideoInvite/build
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+  build-quickstart:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build quickstart
+          command: ./gradlew -q quickstart:assemble
+
+      # Save the builds to workspace
+      - persist_to_workspace:
+          root: .
+          paths:
+            - quickstart/build
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+
+  build-quickstartKotlin:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build quickstartKotlin
+          command: ./gradlew -q quickstartKotlin:assemble
+
+      # Save the builds to workspace
+      - persist_to_workspace:
+          root: .
+          paths:
+            - quickstartKotlin/build
+
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+workflows:
+  version: 2
+
+  build-test-publish:
```

Comment:
Rename `build-test-publish` to just `build-test`. There is no publishing happening in this workflow.